configuration/drugs/drugs.csv

```diff
@@ -1,323 +1,321 @@
-85580d71-7bef-415c-8b88-4168285ace82,,Vitamin C Gouttes 100 mg/ml,b7bab21f-5d84-4e48-b056-bdc25e696365,90a3f930-2eef-4a2e-8427-5359607c7b13,52c5bd1b-7679-45cc-ac8e-1eb8dd92ab7a,52c5bd1b-7679-45cc-ac8e-1eb8dd92ab7a,100 mg/ml,100,95,bahmni_product.categ_saleable_drugs,Stockable Product,b5be6719-64d3-4914-b6d3-8db2e25f1fd4,
-af3fce1f-dcb1-4f76-ad4c-b8ebda43070c,,Vitamine B-Complex Injection,836b7f4c-9e3c-4a53-bb08-ab8990b5b9d8,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,f303ec09-ab78-43d9-abf2-06839f4eb491,f303ec09-ab78-43d9-abf2-06839f4eb491,N/A,140,125,bahmni_product.categ_saleable_drugs,Stockable Product,57b284c4-5a7e-4624-a1ce-30b6103265f1,
-481e357d-6c45-4c8a-beec-6d8f858f2552,,Vitamine B-Complex Co,836b7f4c-9e3c-4a53-bb08-ab8990b5b9d8,fdbe907b-6abc-41dc-9599-2d84105e4063,74ea4fb4-1b15-4530-98b4-eb21865d1a75,74ea4fb4-1b15-4530-98b4-eb21865d1a75,N/A,65,60,bahmni_product.categ_saleable_drugs,Stockable Product,d8f999db-b606-42fa-8f41-e6b464cbdd3a,
-8a54b957-8c4d-4ab6-9e42-575bf5b7be83,,Vitamin A,54528cf8-0022-4f32-a038-cbb2d5965ae5,90a3f930-2eef-4a2e-8427-5359607c7b13,52c5bd1b-7679-45cc-ac8e-1eb8dd92ab7a,52c5bd1b-7679-45cc-ac8e-1eb8dd92ab7a,N/A,140,120,bahmni_product.categ_saleable_drugs,Stockable Product,9273d79b-2f4e-4fd9-910d-6241858b565d,
-9eb0a661-f27b-472e-a35d-04dc1dba0860,,Mycrogynon 30,b2727136-70f5-470b-81a2-26aeff3f346f,fdbe907b-6abc-41dc-9599-2d84105e4063,74ea4fb4-1b15-4530-98b4-eb21865d1a75,74ea4fb4-1b15-4530-98b4-eb21865d1a75,N/A,800,725,bahmni_product.categ_saleable_drugs,Stockable Product,b99aa649-d92b-44e2-9194-a439076b1520,
-61444a80-77b6-4986-b9c1-07eab731ce28,,Microlut 30,71eb922d-5a51-415a-a838-8a815280abfa,fdbe907b-6abc-41dc-9599-2d84105e4063,74ea4fb4-1b15-4530-98b4-eb21865d1a75,74ea4fb4-1b15-4530-98b4-eb21865d1a75,N/A,110,95,bahmni_product.categ_saleable_drugs,Stockable Product,1c73cc9d-3340-4992-86d5-32078d3ca1ee,
-012f0f5d-c76e-4a28-8e60-30c440f5ff62,,Depo Provera,6533b40f-a783-435f-9c92-19d5e7343d98,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,f303ec09-ab78-43d9-abf2-06839f4eb491,f303ec09-ab78-43d9-abf2-06839f4eb491,N/A,140,125,bahmni_product.categ_saleable_drugs,Stockable Product,c6a1075b-f21b-416f-9e3b-e45a0b641e56,
-62a34f17-e83c-4dbc-b53b-9a7f9478501a,,RHEZ,c103ec05-577d-4f64-85ec-54c361895198,fdbe907b-6abc-41dc-9599-2d84105e4063,74ea4fb4-1b15-4530-98b4-eb21865d1a75,74ea4fb4-1b15-4530-98b4-eb21865d1a75,N/A,135,120,bahmni_product.categ_saleable_drugs,Stockable Product,7e12b8d9-c599-40fd-b89f-df2e7e4e0093,
-667682cc-6731-4784-acdc-c7fb7c98d15a,,RH,606f9339-1430-4a00-8515-55095ef66c3b,fdbe907b-6abc-41dc-9599-2d84105e4063,74ea4fb4-1b15-4530-98b4-eb21865d1a75,74ea4fb4-1b15-4530-98b4-eb21865d1a75,N/A,135,115,bahmni_product.categ_saleable_drugs,Stockable Product,ab3c5027-c77f-4207-89a0-dea08dd665a2,
\ No newline at end of file
+Uuid,Void/Retire,Name,Concept Drug,Concept dosage form,Strength,_order:1000

```

Comment:
_Order is not neccessary.
Revert to _version.
Let's keep it as 'Concept Dosage Form'